pages/test_analytics/importing_json.md.erb

```diff
@@ -137,6 +137,7 @@ Schematically, the JSON test results data is like this:
    - [Test result](#json-test-results-data-reference-test-result-objects)
       * [History](#json-test-results-data-reference-history-objects)
          - [Spans](#json-test-results-data-reference-span-objects)
+             * [Detail](#json-test-results-data-reference-detail-objects)
```

Comment:
```suggestion
   - [Test result](#json-test-results-data-reference-test-result-objects)
      * [History](#json-test-results-data-reference-history-objects)
   - [Spans](#json-test-results-data-reference-span-objects)
      * [Detail](#json-test-results-data-reference-detail-objects)
```

Reply:
This should fix the outright weird behavior of the item if that's the levels you want in your list.

Reply:
I want 'spans' to be under history, and 'detail' under 'span'. Seems like 5 levels of nesting is not allowed/working.
If I remove the top level bullet point, it works.

![Screenshot from 2022-08-15 13-22-38](https://user-images.githubusercontent.com/1182969/184563050-a90f7be2-c663-4444-b2c8-976b8da3af92.png)